docs/aerodromes/classd/Moorabbin.md

```diff
@@ -22,48 +23,139 @@ MB ADC is responsible for the Class D airspace in the MB CTR `SFC` to `A025`.
 Refer to [Class D Tower Separation Standards](../../../separation-standards/classd) for more information.
 
 ## Runway Modes
-Equal preference is given to the following runway modes:
 
-- Runways 35L & 35R
-- Runways 17L & 17R
-- Runways 31L & 31R
-- Runways 13L & 13R
+Preferred runway modes are as follows (in order of preference):
 
-When operationally required, runway 04 or 22 may be used, but consideration must be given to it's limited length.
+1. Runway 35L/35R  
+2. Runway 17L/17R  
+3. Runways 13/31 (equal preference)
 
-## VFR Inbound Procedures
+Runway 04/22 may only be used operationally. Consideration must be given to it's limited length.
```

Comment:
Dunno, that's pre-existing text that was there in the old version :)

Usually in real life I know they tend to never use 22 in practicality.

Reply:
Any preference on the wording here. They have never used 22 in my couple years of going in and out. The runway now has warehouses on one side. They tend to favour cross winds for 13/17 etc over ever using this runway. 

17/35 is preferred since it lines up with the tower. Easier to see out which is why I listed it as preferred

Reply:
Just had a discussion with a real world MB controller

He has indicated 
> It can't be nominated, only used on request/requirement 
> Even when xw is over tolerance for all other rwys, i believe 22/04 will still not be nominated
> I've been at MB almost a year, seen it used once on request

I asked what would be a example of a requirement or request:
> Requirement would be emergency/pan, request is basically just shits and gigs
> Outside no traffic random requests, it's only used in emergencies
> Request would only be granted if no traffic
> Runway 04 likely not be used due to vacating on a runway but likely be ok vacate B on request but no-one ever asks for 04 usually

Maybe something like:

Runway 04/22 is not nominated by default and is only available on request or when operationally required for emergencies. Requests for Runway 22 may be granted if traffic conditions allow. Runway 04 is less likely to be approved due the need to vacate onto an intersecting runway.